roads.mss

```diff
@@ -1279,93 +1279,15 @@
   [type='pedestrian']   { marker-transform: translate(-0.5 * @rdz18_pedestrian - 0.5 * @rdz18_cycle); }
 }
 
-
-// ---- Bridge fill for dashed lines --------------------------------
-#tunnel::line[zoom>=14],
-#bridge::line[zoom>=14] {
```

Comment:
I'm fine with this for tunnel. But isn't it an issue for bridge?

Reply:
No, roads have plain inline color